financial-connections/src/main/java/com/stripe/android/financialconnections/repository/CoreAuthorizationPendingNetworkingRepairRepository.kt

```diff
@@ -26,6 +26,6 @@ internal class CoreAuthorizationPendingNetworkingRepairRepository @Inject constr
 
     @Parcelize
     data class State(
-        val coreAuthorization: String? = null
```

Comment:
Removing the nullability here, since having a non-null state should be an indication that a networking relink flow is underway.